crates/cosmos/cosmos-test-components/src/types/dynamic_gas_config.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+pub struct DynamicGasConfig {
+    pub multiplier: f64,
+    pub max: f64,
```

Comment:
Actually, I think we can just put the EIP query type here, so we don't need extra getter for it.